moodlecli/utils.py

```diff
@@ -150,73 +150,28 @@ def setup_duplicate_course(
     moodle_client, base_course_id, coursedata, instructor_role_id,
     student_role_id
 ):
-    """Setup a new course using a base and data from bulk CSV"""
```

Comment:
Minor: This was probably accidental, but the function docstring here was moved into the function implementation. Let's move that back up here.